private-registry-helm-values/values-private-container-registry.yaml

```
@@ -2,36 +2,36 @@
 # Kubecost has monthly releases and often time point releases within the month to address any issues.
 # To get a dynamic list of current images, run the following script:
 # install yq with brew / pip / etc
-# helm template kubecost kubecost/cost-analyzer --set networkCosts.enabled=true --set global.thanos.enabled=true --set prometheus.kubeStateMetrics.enabled=false | yq '..|.image? | select(.)' | sort -u
+# helm template kubecost --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer --set networkCosts.enabled=true --set global.thanos.enabled=true --set prometheus.kubeStateMetrics.enabled=false --set clusterController.enabled=true | yq '..|.image? | select(.)' | sort -u
 # If you don't have yq this should output the same:
-# helm template kubecost kubecost/cost-analyzer --set networkCosts.enabled=true --set global.thanos.enabled=true --set prometheus.kubeStateMetrics.enabled=false | grep 'image:'|sed 's/image: //g'|sed 's/"//g'|sed 's/- //g'|sed 's/ //g'|sort -u
+# helm template kubecost --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer --set networkCosts.enabled=true --set global.thanos.enabled=true --set prometheus.kubeStateMetrics.enabled=false --set clusterController.enabled=true | grep 'image:'|sed 's/image: //g'|sed 's/"//g'|sed 's/- //g'|sed 's/ //g'|sort -u
 # if you use docker to pull images, you can use the output to pull the images:
 # (previous_command)| xargs -I {} docker pull {}
 
 # Images from https://github.com/kubecost/cost-analyzer-helm-chart/tree/develop/cost-analyzer
 # imagePullSecrets:
 # - name: "image-pull-secret"
 
-imageVersion: "prod-1.98.0" # If set, overrides default of $.Chart.AppVersion. Ignored if setting fullImageName for each Kubecost image.
+imageVersion: "prod-1.104.0" # If set, overrides default of $.Chart.AppVersion. Ignored if setting fullImageName for each Kubecost image.
 
 kubecostFrontend:
-  # fullImageName: "YOUR_REGISTRY/frontend:prod-1.91" # Use image -OR- fullImageName.
+  # fullImageName: gcr.io/kubecost1/frontend:prod-1.104.0 # Use image -OR- fullImageName
```

This is the only one that was updated to reflect an actual image path. Should we update all of them or prevent this one back to `YOUR_REGISTRY`

```suggestion
  # fullImageName: YOUR_REGISTRY/frontend:prod-1.104.0 # Use image -OR- fullImageName
```